build/llamafile-convert

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+FILE=$1
+TYPE=${2:-both}
+SCRIPTNAME=$(basename $0)
+
+if [ -z "$FILE" ]; then
+  echo "Usage: $SCRIPTNAME <gguf file or url> [cli|server|both]"
+  exit 1
+fi
+
+# if the file starts with http
+if [[ $FILE == http* ]]; then
+  # download the file
+  # if the filename contains ?download=true, remove it
+  FILE=$(echo $FILE | sed 's/?download=true//g')
+  # get the filename
+  FILENAME=$(echo $FILE | sed 's/.*\///g')
+  echo "Downloading $FILENAME"
+  wget -O $FILENAME $FILE
```

Comment:
Some stock systems don't have `wget` but have `curl`. With other systems, it's the other way around. I recommend copying and pasting the downloader detection code I wrote in this script:

https://github.com/Mozilla-Ocho/llamafile/blob/586b40877b74abfbc5e4df3918ee4595afb4a2d6/build/download-cosmocc.sh#L55-L65

You can then say:

```sh
"${DOWNLOAD}" ${DOWNLOAD_ARGS} "$FILENAME" "$FILE"
```